### PR TITLE
Edit artisan deletes picture

### PR DESCRIPTION
**Pull request by smrizzo**
Fixed issue where edit artisan is deleting images after not changing photo.

### CODE REVIEW

File: src/screens/ArtisanHub/actions.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import firebase from 'react-native-firebase'
+import { regExpLiteral } from '@babel/types';
```
Comment (BryceV) [inline src/screens/ArtisanHub/actions.js]: Run lint to remove these semicolons

**Review by BryceV — COMMENTED** [review]
Testing it now. But do you know why it wouldnt work before? If you deleted an image from firebase and then re-uploaded it, it shouldn't have created a problem.

**Comment by smrizzo** [comment]
Yes I explained the reason it was doing that in the jira ticket.

File: src/screens/ArtisanHub/actions.js (hunk 1 above)
Comment (smrizzo) [reply src/screens/ArtisanHub/actions.js]: Fixed linter issues

**Review by BryceV — APPROVED** [review]
Looks good!

**Pull request merged**